packages/bpk-component-radio/src/BpkRadio.js

```diff
@@ -46,6 +64,7 @@ const BpkRadio = props => {
         name={name}
         disabled={disabled}
         aria-label={ariaLabel || label}
+        aria-invalid={isInvalid}
```

Comment:
This should only be set if valid is defined. With this code, if `valid` is undefined or null this will set `aria-invalid="false"` which isn't necessarily true.

Reply:
@georgegillams I think this is protected against by having the explicit check above (https://github.com/Skyscanner/backpack/pull/1589/files/72f1851552272704d0ebb3e082fd8d1e15fad6fc#diff-ff05dd3a3cbb09216a69f345e8cc07ccR40)
This is taken from the BpkInput code.

Reply:
Just checked the logic and it's 💯  👍